Gemfile

```diff
@@ -1,8 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'rubygems-tasks'
+gemspec
+
 gem 'git'
 gem 'rake'
 gem 'rubocop', "~> 0.39.0" # in line with puppet gem and its rubocop.yml we're using
```

Comment:
Shouldn't it be moved to gem spec file, same with git and rake gems.

Reply:
Yep, good call, fixed